src/Native/pkg/runtime.native.System/runtime.native.System.builds

```diff
@@ -3,9 +3,15 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="runtime.native.System.pkgproj"/>
-    <Project Condition="'$(TargetsCentOS)' == 'true'" Include="centos\runtime.native.System.pkgproj"/>
-    <Project Condition="'$(TargetsLinux)' == 'true'" Include="ubuntu\runtime.native.System.pkgproj"/>
-    <Project Condition="'$(TargetsOSX)' == 'true'" Include="osx\runtime.native.System.pkgproj"/>
+    <Project Condition="'$(TargetsCentOS)' == 'true'" Include="centos\runtime.native.System.pkgproj">
+      <AdditionalProperties>Platform=amd64</AdditionalProperties>
```

Comment:
I suspect https://github.com/dotnet/corefx/blob/master/dir.traversal.targets#L8 is the best place for this.